package.json

```diff
@@ -50,5 +50,10 @@
     "extends": [
       "cozy"
     ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn doc"
```

Comment:
Is it fast enough ? Maybe we could do that in prepush ? And fail if the staging area has something ?

Reply:
Shit that wasn't supposed to be part of this PR, especially because it doesn't work :'). BRB removing it.